src/resources/postcss/tickets-refresh.pcss

```diff
@@ -210,6 +255,27 @@
 	}
 }
 
+/* Adjust font size for entire form */
+#event_tickets {
+	font-size: 12px;
```

Comment:
Setting a base font size for the component (in px - could use rem?) that everything inside scales from.